frontend/pulse3d/src/components/layouts/DashboardLayout.js

```diff
@@ -15,11 +16,15 @@ export const UploadsContext = createContext();
 export default function DashboardLayout({ children }) {
   const [uploads, setUploads] = useState([]);
   const [fetchUploads, setFetchUploads] = useState(false);
+  const [pulse3dVersions, setPulse3dVersions] = useState([]);
   const router = useRouter();
 
   useEffect(() => {
     if (router.pathname === "/uploads") {
       getUploads();
+      getPulse3dVersions();
```

Comment:
The dashboard layout it always loaded right? If so, we could probably just hit this route and save the versions once, right when it gets loaded. Or if not, we could just check if `pulse3dVersions` and only hit the route if it's an empty list

Reply:
it refreshes though when the router changes at the moment which I think happens between the uploads page and the uploads-form but I could be wrong..